<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-U67QH3E" ignored="0" name="wangdonghui" tests="25" failures="4" timestamp="07 十二月 2018 01:53:41 GMT" time="225.885" errors="0">
  <testcase name="openBrower" time="3.604" classname="com.webtest.xinhu_personal.Login"/>
  <testcase name="login" time="3.436" classname="com.webtest.xinhu_personal.Login"/>
  <testcase name="login" time="3.371" classname="com.webtest.xinhu_personal.Login"/>
  <testcase name="login" time="3.381" classname="com.webtest.xinhu_personal.Login"/>
  <testcase name="login" time="3.363" classname="com.webtest.xinhu_personal.Login"/>
  <testcase name="loginsuccess" time="3.376" classname="com.webtest.xinhu_personal.Login"/>
  <testcase name="login4" time="6.704" classname="com.webtest.xinhu_personal.Addnotice_time"/>
  <testcase name="enterNotice" time="6.538" classname="com.webtest.xinhu_personal.Addnotice_time"/>
  <testcase name="testTime" time="16.433" classname="com.webtest.xinhu_personal.Addnotice_time"/>
  <testcase name="testTime" time="16.422" classname="com.webtest.xinhu_personal.Addnotice_time">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.xinhu_personal.Addnotice_time.testTime(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testTime -->
  <testcase name="testTime" time="16.374" classname="com.webtest.xinhu_personal.Addnotice_time">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.xinhu_personal.Addnotice_time.testTime(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testTime -->
  <testcase name="login1" time="6.714" classname="com.webtest.xinhu_personal.AddrList"/>
  <testcase name="addrList" time="9.775" classname="com.webtest.xinhu_personal.AddrList"/>
  <testcase name="searchList" time="3.313" classname="com.webtest.xinhu_personal.AddrList"/>
  <testcase name="searchList" time="3.333" classname="com.webtest.xinhu_personal.AddrList"/>
  <testcase name="searchList" time="3.347" classname="com.webtest.xinhu_personal.AddrList"/>
  <testcase name="login2" time="6.738" classname="com.webtest.xinhu_personal.Addnotice"/>
  <testcase name="enterNotice" time="6.546" classname="com.webtest.xinhu_personal.Addnotice"/>
  <testcase name="addNotice" time="19.637" classname="com.webtest.xinhu_personal.Addnotice">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.xinhu_personal.Addnotice.addNotice(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addNotice -->
  <testcase name="addNotice" time="19.638" classname="com.webtest.xinhu_personal.Addnotice">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.xinhu_personal.Addnotice.addNotice(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addNotice -->
  <testcase name="addNotice" time="19.647" classname="com.webtest.xinhu_personal.Addnotice"/>
  <testcase name="login3" time="6.845" classname="com.webtest.xinhu_personal.ReInfor"/>
  <testcase name="enterInfor" time="9.835" classname="com.webtest.xinhu_personal.ReInfor"/>
  <testcase name="reInfor1" time="3.296" classname="com.webtest.xinhu_personal.ReInfor"/>
  <testcase name="reInfor2" time="0.018" classname="com.webtest.xinhu_personal.ReInfor"/>
</testsuite> <!-- wangdonghui -->
